frontend/src/pages/HomePage/index.js

```diff
@@ -16,9 +16,8 @@ const HomePage = () => {
           </div>
 
           <div className={styles.headingWrapper}>
-            <Heading2>
-              Register to play and<br></br> win a prize!
-            </Heading2>
+            <Heading2>Register to play and</Heading2>
+            <Heading2>win a prize!</Heading2>
```

Comment:
Hmm. Nice try. But nope. 🙂

Reply:
I tried with {"\n"} with {"aa\naa"} and with <br></br>. The only one that worked was the last one but it says I'm passing an array to Heading2

Reply:
Tried something I found on google about setting white-space: pre-line; and then using \n. Is that good?

Reply:
Is this going to be solved with the `max-width` styles in your other commit?

Reply:
It's not but it can be solved in that way too, so I'll change it!